src/OpenTelemetry.Shims.OpenTracing/SpanShim.cs

```diff
@@ -45,11 +45,11 @@ internal sealed class SpanShim : global::OpenTracing.ISpan
 
         public SpanShim(TelemetrySpan span)
         {
-            this.Span = span ?? throw new ArgumentNullException(nameof(span));
+            this.Span = span ?? throw new ArgumentNullException(nameof(span), "Parameter cannot be null");
```

Comment:
should we end the text with a .?

Reply:
**https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/exceptions/creating-and-throwing-exceptions**
From this guidelines, there is no "." ending.